source/common/router/header_parser.h

```diff
@@ -1,3 +1,11 @@
+// grpcpp/grpcpp.h includes <windows.h>, so undef some interfering symbols
```

Comment:
?? this seems a wrong place.

Reply:
I mean this could never go before #pragma once, and grpcpp is not included in this file either.

Reply:
Agreed. Tested and removing.